app/src/app/components/header/Header.tsx

```diff
@@ -22,13 +23,17 @@ export default function Header() {
                 Packit
               </div>
             </NavLink>
-            <div className="mx-3 flex items-center md:hidden">
-              <HeaderMenu />
-            </div>
-            {isAuthenticated && <LeftNav className="mx-6 hidden md:flex" />}
+            {isAuthenticated && (
+              <>
+                <div className="mx-3 flex items-center md:hidden">
+                  <NavMenuMobile />
+                </div>
+                <LeftNav className="mx-6 hidden md:flex" />
+              </>
+            )}
```

Comment:
I'm not sure you want to exclude the whole NavMenuMobile if not authenticated, because it also includes Accessibility which should always be visible.

Reply:
good catch thanks sorted